Commands/Server/Server.go

```diff
@@ -55,19 +56,25 @@ func Run(args []string) {
 
 	go rpcServer.Start(forkliftRpc)
 
+	var currentPath, err = GetCurrentExecPath()
+	if err != nil {
+		logger.Fatalf("Failed to get current executable path: %s", err)
+		os.Exit(1)
```

Comment:
No need, the code will terminate if there's an error anyway.

Reply:
its custom logger, it will not terminate on `.Fatal`